### PR TITLE
Team

### PR DESCRIPTION
**Pull request by zachlp2016**
Creating pull request to merge team branch.

### CODE REVIEW

File: test/team_test.rb [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+                            :franchise_id=>"14",
+                            :short_name=>"Los Angeles",
+                            :team_name=>"Kings",
+                            :abbreviation=>"LAK",
+                            :link=>"/api/v1/teams/26"
+                            }
+
+    @team_1 = Team.new(@team_data_1)
+    @team_2 = Team.new(@team_data_2)
+    @team_3 = Team.new(@team_data_3)
```
Comment (nagerz) [inline test/team_test.rb]: These look nice. Will want to add a couple more for the league stats stuff I have been writing

File: test/team_test.rb [hunk 2]
```diff
@@ -0,0 +1,53 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/team'
+require 'pry'
+
+class TeamTest < Minitest::Test
+
+  def setup
+    @team_data_1 = {
```
Comment (PeregrineReed) [inline test/team_test.rb]: Can the indentation be fixed here?

File: test/team_test.rb [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+                            :team_id=>"1",
+                            :franchise_id=>"23",
+                            :short_name=>"New Jersey",
+                            :team_name=>"Devils",
+                            :abbreviation=>"NJD",
+                            :link=>"/api/v1/teams/1"
+                            }
+
+    @team_data_2 = {
```
Comment (PeregrineReed) [inline test/team_test.rb]: Can the indentation be fixed here?

File: test/team_test.rb [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+  end
+
+  def test_it_has_attributes
+    assert_equal "1", @team_1.team_id
+    assert_equal "23", @team_1.franchise_id
+    assert_equal "New Jersey", @team_1.short_name
+    assert_equal "Devils", @team_1.team_name
+    assert_equal "NJD", @team_1.abbreviation
+    assert_equal "/api/v1/teams/1", @team_1.link
+  end
```
Comment (nagerz) [inline test/team_test.rb]: Nice job, looks good.

File: test/team_test.rb [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+    @team_data_2 = {
+                            :team_id=>"4",
+                            :franchise_id=>"16",
+                            :short_name=>"Philadelphia",
+                            :team_name=>"Flyers",
+                            :abbreviation=>"PHI",
+                            :link=>"/api/v1/teams/4"
+                            }
+
+    @team_data_3 = {
```
Comment (PeregrineReed) [inline test/team_test.rb]: Can the indentation be fixed here?

**Pull request merged**